README.md

```diff
@@ -114,3 +114,60 @@ server {
 2. Run `nginx -t` to make sure, that your config is valid
 3. Run `systemctl restart nginx` (or equivalent) to restart your nginx and apply the new settings
 4. Your nginx ui is now accessible at nginx.mydomain.com and will correctly prompt for basic auth
+
+### Example NginX-UI + NginX (run in container) behind path '/some-location'
+
+docker-compose.yml
+```yaml
+version: '3'
+services:
+  nginx-ui:
+    container_name: nginx-ui
+    build: .
+    image: schenkd/nginx-ui:latest
+    # ports:
+    #   - 8080:8080
+    volumes:
+      - D:/_data/_docker/nginx:/etc/nginx
```

Comment:
Could you adjust the volume mounts to a unix path? I don't want to get requests related to windows support since I don't have a windows machine where I could test it. Thanks mate!